stdlib/experimental/query/from.flux

```diff
@@ -0,0 +1,46 @@
+package query
+
+fromRange = (bucket, start, stop=now()) =>
+    from(bucket: bucket)
+        |> range(start: start, stop: stop)
+
+filterMeasurement = (table=<-, measurement) => table |> filter(fn: (r) => r._measurement == measurement)
+
+filterFields = (table=<-, fields=[]) =>
+    if length(arr: fields) == 0 then
+        table
+    else
+        table |> filter(fn: (r) => contains(value: r._field, set: fields))
+
+inBucket = (
+    bucket,
+    measurement,
+    start,
+    stop=now(),
+    fields=[],
+    predicate=(r) => true
+) =>
+    fromRange(bucket: bucket, start: start, stop: stop)
+        |> filterMeasurement(measurement)
+        |> filter(fn: predicate)
+        |> filterFields(fields)
+
+
+// counter funcs
+_rateDet = (table=<-, unit=0s, nonNegative=true) =>
+    if string(v: unit) == "0s" then
+        table |> difference(nonNegative: nonNegative, columns: ["_value"])
+    else
+        table |> derivative(unit: unit, nonNegative: nonNegative, columns: ["_value"], timeColumn: "_time")
+
+_sumWindowDet = (table=<-, window=0s, fn=mean) =>
+    if string(v: window) == "0s" then
+        table
+    else
+        table |> aggregateWindow(every: window, fn: fn)
+
+aggregateCounter = (table=<-, pollingInterval=0s, window, groupCols=[]) =>
+    table
+        |> _rateDet(unit: pollingInterval, nonNegative: true)
```

Comment:
What is the `pollingInterval`? The polling interval should not be needed to correctly understand counters.